apache/superset
# feat: Cross-referenced Dashboards in Chart list (Column + Filter)

### SUMMARY
This PR adds the ability to filter the Charts by the cross-referenced Dashboards. It also shows the referenced Dashboards in a new column.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

https://user-images.githubusercontent.com/60598000/194886095-2a8e25a3-0b37-42c0-8212-ad3a1d1feb16.mp4

### TESTING INSTRUCTIONS
1. Filter the Charts list by a Dashboard and make sure the list is filtered correctly
2. Add a Chart to multiple Dashboards, make sure the references are appearing in the "Dashboards added to" column
3. Add more Dashboards than what fits the "Dashboards added to"  column. Make sure the tooltip is appearing
4. Add more than 20 Dashboards. Make sure the "Plus X more" is appearing in the tooltip

### ADDITIONAL INFORMATION
